WebContent/WEB-INF/jsp/controlPanel.jsp

```diff
@@ -143,7 +143,7 @@
 									   <span class="submenuText"><input type="radio" name="size" value="2" > Service (768px)</span><br />
 									   <span class="submenuText"><input type="radio" name="size" value="6" > Large (1200px)</span><br />
 									   <span class="submenuText"><input type="radio" name="size" value="7" > Huge (1600px)</span><br />
-									   <span class="submenuText"><input type="radio" name="size" value="v" > Videos mp4 (720p) </span><br />
+									   <span class="submenuText"><input type="radio" name="size" value="v" > MP3/MP4 </span><br />
```

Comment:
Done! It's in a separate commit now, https://github.com/ucsdlib/damsmanager/commit/b3302eb865388447155ee75a926622e22dc9c57d.